server/container_create.go

```diff
@@ -56,6 +57,60 @@ func addOciBindMounts(sb *sandbox, containerConfig *pb.ContainerConfig, specgen
 	return nil
 }
 
+// buildOCIProcessArgs build an OCI compatible process arguments slice.
+func buildOCIProcessArgs(containerKubeConfig *pb.ContainerConfig, imageOCIConfig *v1.Image) []string {
+	processArgs := []string{}
+
+	kubeCommands := containerKubeConfig.Command
+	kubeArgs := containerKubeConfig.Args
+
+	if imageOCIConfig == nil {
+		// We did not get an OCI Image configuration.
+		// We should only use the information we got from kubelet,
+		// and fall back to a reasonable default otherwise.
```

Comment:
@runcom So the logic would be:

* Erroring out if the image config is missing
* If we have an image config, we'd only use it for filling the gaps we'd potentially get from the kubelet conf

I'm fine with that.